### PR TITLE
tox + travis-ci

### PR DESCRIPTION
**Pull request by steenzout**
- use [tox](http://tox.readthedocs.io) to setup test environments
- added [travis-ci](https://travis-ci.org/) integration

I'll provide shortly another PR to fix all lint'ing issues.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist = py{36,35,34,33,27,26}
```
Comment (kxepal) [inline tox.ini]: But py26 is missed from travis matrix...

**Comment by kxepal** [comment]
I don't know reasons why, but thanks you a lot for making things better! 
Does it green with  all those pep8 and else checks?

File: tox.ini (hunk 1 above)
Comment (steenzout) [inline tox.ini]: I'll add that.

**Comment by steenzout** [comment]
@kxepal I had to remove one PEP8 check (it wants imports on top-level of module).

I can patch the module to be compliant with the PEP8 directive but that would require a bit more code changes than I want to do now.

the tests pass now: [link](https://travis-ci.org/steenzout/python-viivakoodi/builds/155981509).

let me know if you want me to squash the commits and if there is anything you don't agree with.

File: tox.ini (hunk 1 above)
Comment (steenzout) [inline tox.ini]: added.

**Comment by kxepal** [comment]
No worries. Thanks a lot!

**Pull request merged**